lib/Query/Resolver.php

```diff
@@ -264,14 +268,52 @@ public function escapeIdentifier($identifier)
         return $this->mapper->connection()->quoteIdentifier(trim($identifier));
     }
 	
-	/**
-	 * Trim a leading schema name separated by a dot if present
-	 *
-	 * @param string $identifier
-	 * @return string
-	 */
-	public function trimSchemaName($identifier){
-		$components = explode('.', $identifier, 2); 
-		return end($components);
-	}
+    /**
+     * Trim a leading schema name separated by a dot if present
+     *
+     * @param string $identifier
+     * @return string
+     */
+    public function trimSchemaName($identifier){
+            $components = explode('.', $identifier, 2);
+            return end($components);
+    }
+
+    /**
+     * Add foreign keys from BelongsTo relations to the table schema
+     * @param Table $table
+     * @return Table
+     */
+    protected function addForeignKeys(Table $table)
+    {
+        $relations = $this->mapper->entityManager()->relations();
```

Comment:
The $relations variable is empty for me. Could somebody look into this bug?
I replaced the mentioned line with this and now it works fine:
```php
$entityName = $this->mapper->entity();
$relations = $entityName::relations($this->mapper, new $entityName);
```